src/pages/App.tsx

```diff
@@ -119,29 +119,6 @@ const Marginer = styled.div`
   margin-top: 5rem;
 `
 
-function getCurrentPageFromLocation(locationPathname: string): PageName | undefined {
```

Comment:
A lot of these pages have different variants, which either are not covered, or should probably be covered under a different name.

As part of removing the trace in App, could you audit these components, and determine if they should have unique PageName's for analytics?
- `/vote`: there are separate `Landing`, `VotePage`, and `CreateProposal` pages, but they all use `PageName.VOTE_PAGE`.
- `/pool`: there are separate `PoolFinder`, `PoolV2`, `Pool`, and `PositionPage` pages, but they all use `PageName.POOL_PAGE`.
- `/tokens`: there are separate `Tokens` and `TokenDetails` pages, but they both use `PageName.TOKENS_PAGE`.
- `PageName.NFT_COLLECTION_PAGE`: this is included through `Collection`, but also added through `ActivityCells.tsx` and `CollectionAsset.tsx`. The latter two should be removed.